core/domain/change_domain.py

```diff
@@ -229,7 +237,9 @@ def to_dict(self) -> Dict[str, str]:
 
         return base_change_dict
 
-    def __getattr__(self, name: str) -> str:
+    # Here we use Any, because getattr is of Callable type and
+    # it does not make sense to check it against anything.
```

Comment:
The comment doesn't make sense, the problem is not that `getattr` is callable, every function is callable and we still check them. The problems is that the change_domain can have a differing structure and we cannot know what exactly it will return.

Reply:
Fixed and pushed.